apps/safe-claiming-app/src/widgets/ClaimingWidget.tsx

```diff
@@ -140,12 +148,11 @@ const ClaimingWidget = () => {
     <>
       <div>
         <Title>Your voting power</Title>
-        <Box
-          display="flex"
-          alignItems="center"
-          justifyContent="center"
-          gap={1}
-          mb={1}
+        <StyledButtonLink
+          href={claimingSafeAppUrl}
+          rel="noopener noreferrer"
```

Comment:
I would remove this, we will then see in the analytics how many people clicked on this.

Reply:
I will remove it then.
Also the CI failed as it does not allow me to pass the `<a>` props to the button even after passing the `href` .... I'm finding a solution